src/components/MultiHopTrade/components/TradeConfirm/TradeConfirm.tsx

```diff
@@ -150,6 +150,22 @@ export const TradeConfirm = () => {
 
   const txHash = buyTxHash ?? sellTxHash
 
+  const priceImpactPercentage = useMemo(() => {
+    if (!sellAmountBeforeFeesUserCurrency || !netBuyAmountUserCurrency) return bn(0)
+
+    const tradeDifference = bn(sellAmountBeforeFeesUserCurrency)
+      .minus(netBuyAmountUserCurrency)
```

Comment:
It feels a little weird that we include the buy asset protocol fee here, as this has nothing to do with the price impact of the trade due to the size of the pool.

Including this thus makes the popup a little misleading, as it can who when there is a large buy asset protocol fee relative to the size of the trade.

Reply:
Used before fees amount in https://github.com/shapeshift/web/pull/5032/commits/56499954f15e6b28918e49d5ffb194a427f34a33.